scripts/create_polygons.py

```diff
@@ -51,6 +52,9 @@ def main() -> None:  # pylint: disable=too-many-locals
     output_files = []
 
     for file in source:
+        if not is_tiff(file):
+            get_log().error("create_polygon_file_not_tiff_skipped", file=file)
```

Comment:
Should these be info/debug rather than an error. 
I'm not sure but I feel like we'll be expecting to skip a number of files :thinking:

Reply:
Right, even `trace` would be best